src/hotspot/os_cpu/windows_x86/os_windows_x86.cpp

```diff
@@ -56,68 +56,22 @@
 #undef REG_SP
 #undef REG_FP
 #undef REG_PC
-#ifdef AMD64
 #define REG_SP Rsp
 #define REG_FP Rbp
 #define REG_PC Rip
-#else
-#define REG_SP Esp
-#define REG_FP Ebp
-#define REG_PC Eip
-#endif // AMD64
 
 JNIEXPORT
 extern LONG WINAPI topLevelExceptionFilter(_EXCEPTION_POINTERS* );
 
 // Install a win32 structured exception handler around thread.
 void os::os_exception_wrapper(java_call_t f, JavaValue* value, const methodHandle& method, JavaCallArguments* args, JavaThread* thread) {
   __try {
-
-#ifndef AMD64
-    // We store the current thread in this wrapperthread location
-    // and determine how far away this address is from the structured
-    // exception pointer that FS:[0] points to.  This get_thread
-    // code can then get the thread pointer via FS.
-    //
-    // Warning:  This routine must NEVER be inlined since we'd end up with
-    //           multiple offsets.
-    //
-    volatile Thread* wrapperthread = thread;
-
-    if (os::win32::get_thread_ptr_offset() == 0) {
```

Comment:
I think `os::win32::get_thread_ptr_offset` is not needed now and  ./os_cpu/windows_x86/assembler_windows_x86.cpp looks like it can be deleted.

Reply:
I just redisovered this by myself from your previous comment. :) However, there were some more `thread_ptr_offset` I could remove.

`assembler_windows_x86.cpp` is heavily cut down, but can't be fully removed since it contains the Windows implementation of `MacroAssembler::int3()`.